### DISCUSSION TITLE
更新后面板一直提示正在启动中请稍候

### DISCUSSION THREAD

**Discussion by zuoyou0119** [whyour/qinglong#1919] [Q&A]
ql update更新无法正常启动

**Comment by whyour** [comment]
> ql update更新无法正常启动

docker run --rm -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower -cR <容器名>
更新一次

**Reply by zuoyou0119** [reply]
大哥，删库了？？？

**Reply by zuoyou0119** [reply]
得了，直接重新建容器了。感谢大佬让我破釜沉舟
